attesters/hydra-s1/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./hydra-s1-attester";
```

Comment:
The attesters/hydra-s1 and attesters/hydra-s1-simple should not be at the same level as they don't have the same "nature" of content and the hydra-s1-simple is a subtype of hydra-s1

Reply:
I agress. For the moment, I moved to `root`, tell me what you prefer

Reply:
Does not make sens at the root.... 
What about putting the /hydra-s1 folder on 
/attesters/base/hydra-s1
and keeping the /attesters/hydra-s1-simple like this

Reply:
done!